patroni/dcs/consul.py

```diff
@@ -135,6 +135,9 @@ def connect(self, *args, **kwargs):
             kwargs['token'] = self._token
         return HTTPClient(**kwargs)
 
+    def set_token(self, token):
+        self._token = token
+        self._client.http.token = self._token
```

Comment:
According to the https://github.com/cablehead/python-consul/blob/master/consul/base.py#L326 here should be `self.http.token = token`

Reply:
`flake8` doesn't like your formatting: https://travis-ci.org/zalando/patroni/jobs/518699102#L1227

Reply:
Fixed both